vms/platformvm/utxo/handler.go

```diff
@@ -528,9 +532,17 @@ func (h *handler) VerifySpend(
 			)
 		}
 		utxos[index] = utxo
+		signers[index], err = utxoDB.GetMultisigUTXOSigners(utxo)
```

Comment:
Can't we move this part out somehow out of the avax file?
IIRC we have already a camino specific UTXO handler file...